libc/utils/MPFRWrapper/MPFRUtils.h

```diff
@@ -77,6 +77,7 @@ enum class Operation : int {
   Hypot,
   Pow,
   EndBinaryOperationsSingleOutput,
+  Fmul,
```

Comment:
it should be between `BeginBinaryOperationsSingleOutput` and `EndBinaryOperationsSingleOutput`